cg/server/api.py

```diff
@@ -467,6 +470,17 @@ def get_application_pipeline_limitations(tag: str):
     return jsonify([limitation.to_dict() for limitation in application_limitations])
 
 
+@BLUEPRINT.route("/orders")
```

Comment:
Forgot to say, but I would a test for the endpoint like [this](https://github.com/Clinical-Genomics/cg/blob/master/tests/server/endpoints/test_delivery_message_endpoint.py), if you have the energy. It should be pretty straightforward since we have one test doing exactly this already.

Reply:
Added tests